build-android/build.sh

```diff
@@ -40,7 +40,7 @@ if [ "${CLASSICAL}" == "1" ]; then
   $SCONS platform=android arch=arm32 $OPTIONS target=editor store_release=${store_release}
   $SCONS platform=android arch=arm64 $OPTIONS target=editor store_release=${store_release}
   $SCONS platform=android arch=x86_32 $OPTIONS target=editor store_release=${store_release}
-  $SCONS platform=android arch=x86_64 $OPTIONS target=editor store_release=${store_release}
+  $SCONS platform=android arch=x86_64 $OPTIONS target=editor store_release=${store_release} separate_debug_symbols=yes # Generate native debug symbols.
```

Comment:
Is it intended for `separate_debug_symbols=yes` to be only present for the last architecture being built? Does it create debug symbols for all architectures at once? I thought symbols were architecture-specific.

Reply:
It's intended to be last because all it does is zip the directory `platform/android/java/lib/libs/tools/release/` containing the `.so` files generated by each architecture:
- `platform/android/java/lib/libs/tools/release/arm64-v8a` for arm64
- `platform/android/java/lib/libs/tools/release/armeabi-v7a` for arm32
- ...